flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/table/JdbcRowDataLookupFunction.java

```diff
@@ -147,7 +147,7 @@ public void eval(Object... keys) {
 			}
 		}
 
-		for (int retry = 1; retry <= maxRetryTimes; retry++) {
+		for (int retry = 0; retry <= maxRetryTimes; retry++) {
```

Comment:
`sink.max-retries` configuration in `JdbcBatchingOutputFormat` is similar with this, could you also have a change?

BTW, Commit message should contain issue id etc, following 
[https://flink.apache.org/contributing/code-style-and-quality-pull-requests.html](https://flink.apache.org/contributing/code-style-and-quality-pull-requests.html)

Reply:
Thanks for your review.